proxy/codec.go

```diff
@@ -61,8 +55,8 @@ func (c *rawCodec) Unmarshal(data []byte, v interface{}) error {
 	return nil
 }
 
-func (c *rawCodec) String() string {
-	return fmt.Sprintf("proxy>%s", c.parentCodec.String())
+func (c *rawCodec) Name() string {
+	return c.parentCodec.Name()
```

Comment:
I think we should preserve the "proxy->%s" naming here.

Reply:
Just a note, this will not allow it to be used as a regular codec:

> When specified in either of these ways, messages will be encoded using this codec and sent along with headers indicating the codec (`content-type` set to `application/grpc+<codec name>`). 

https://github.com/grpc/grpc-go/blob/v1.65.0/Documentation/encoding.md#using-a-codec